api/v1beta1/activemqartemis_types.go

```diff
@@ -373,6 +373,17 @@ type DeploymentPlanType struct {
 	// Specifies the Revision History Limit of the statefulset
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Revision History Limit"
 	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
+	// Specifies additional Volumes to be attached to the broker pods
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Volumes"
+	ExtraVolumes []corev1.Volume `json:"extraVolumes,omitempty"`
+	// Specifies mount options for extraVolumes
+	// +optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Mount Options"
+	ExtraVolumeMounts []*corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
+	// Specifies External PersistentVolumeClaims for the broker pod
+	// the key is mount path
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="External PVCs"
+	PersistentVolumeClaims map[string]corev1.PersistentVolumeClaimSpec `json:"persistentVolumeClaims,omitempty"`
```

Comment:
Maybe reusing the same field name of the [StatefulSetSpec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#statefulsetspec-v1-apps) struct could help users
```suggestion
	// Specifies Volume Claims Templates for the broker pods
	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Volume Claims Templates"
	VolumeClaimTemplates map[string]corev1.PersistentVolumeClaimSpec `json:"volumeClaimTemplates,omitempty"`
```

Reply:
+1